contracts/src/v0.8/ccip/ocr/MultiOCR3Base.sol

```diff
@@ -115,8 +116,12 @@ abstract contract MultiOCR3Base is ITypeAndVersion, Ownable2StepMsgSender {
 
   uint256 internal immutable i_chainID;
 
+  /// @dev The address used to send calls for gas estimation.
+  address internal immutable i_gasEstimationSender;
+
   constructor() {
     i_chainID = block.chainid;
+    i_gasEstimationSender = address(1);
```

Comment:
can you include a comment as to why this is being set as `address(1)` please.

Reply:
Why do we use a hard coded value to set an immutable value? Doesn't that mean it's simply a constant?